Makefile

```diff
@@ -1,4 +1,6 @@
-PROFILER_VERSION=3.0
+ifeq ($(PROFILER_VERSION),)
```

Comment:
We can use `PROFILER_VERSION ?= 3.0`, which will set it to 3.0 only when the variable is unset.

Reply:
Right, we can! Done so.